src/lib.rs

```diff
@@ -534,30 +510,30 @@ impl UintPadding {
  */
 struct SequenceData {
```

Comment:
This may need to be `#[repr(C)]` to control layout to achieve the desired false-sharing avoidance